.eslintrc.js

```diff
@@ -0,0 +1,84 @@
+module.exports = {
+	root: false,
+	env: { browser: true, es6: true, node: true },
+	extends: [
+		'plugin:@typescript-eslint/recommended',
+		'plugin:@typescript-eslint/recommended-requiring-type-checking',
+		'plugin:@angular-eslint/recommended',
+		'prettier/@typescript-eslint',
+	],
+
+	parserOptions: {
+		project: ['./tsconfig.eslint.json'],
+		tsconfigRootDir: __dirname,
+		sourceType: 'module',
+		extraFileExtensions: ['.json', '.lock'],
+	},
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint', 'prettier'],
+	rules: {
+		'prettier/prettier': 'warn',
+
+		'lines-between-class-members': 'off',
+		'no-useless-constructor': 'off',
+		'no-restricted-syntax': 'off',
+		'prefer-const': 'off',
```

Comment:
Oh, 100% this should be turned on. @Daniel-Lam will never forgive you otherwise!

Reply:
Yes, definitely use `const` instead of `let` where you can... why not? 😄